service/src/main/java/uk/gov/hmcts/reform/fpl/service/UploadC2DocumentsService.java

```diff
@@ -30,12 +33,18 @@
     private final Time time;
     private final SupportingEvidenceValidatorService validateSupportingEvidenceBundleService;
     private final DocumentUploadHelper documentUploadHelper;
+    private final FeatureToggleService featureToggleService;
 
     public List<Element<C2DocumentBundle>> buildC2DocumentBundle(CaseData caseData) {
         List<Element<C2DocumentBundle>> c2DocumentBundle = defaultIfNull(
             caseData.getC2DocumentBundle(), new ArrayList<>()
         );
 
+        if (featureToggleService.isUploadAdditionalApplicationsEnabled() && !c2DocumentBundle.isEmpty()) {
```

Comment:
&& !c2DocumentBundle.isEmpty() should not be needed.
i personally think that comment is not needed, c2DocumentBundle.sort is self explanatory

Reply:
ok sure makes sense. comment removed and the check :)